### PR TITLE
[WIP] GH131: Added Modules section

### PR DESCRIPTION
**Pull request by gep13**
Relates to #131 

Work is still required here, but wanted to put it up for review.

### CODE REVIEW

File: src/Cake.Web.Core/Content/Modules/Module.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        public string Name { get; set; }
+        public Uri Website { get; set; }
+        public Uri Repository { get; set; }
+        public string Author { get; set; }
+        public string Description { get; set; }
+        public List<string> Categories { get; set; }
+        public PackageDefinition PackageDefinition { get; set; }
+
+        public Uri Url => Repository ?? Website;
+        public bool IsOwnedByAddin => true;
```
Comment (devlead) [inline src/Cake.Web.Core/Content/Modules/Module.cs]: We should add an IsOwnedByModule and so we i.e. can add Modules at
https://github.com/cake-build/website/blob/97fb9986af1f233f87caeb8def3bfeaad4c58225/src/Cake.Web/Views/Dsl/Category.cshtml#L35

Comment (gep13) [inline src/Cake.Web.Core/Content/Modules/Module.cs]: @devlead Interesting. I hadn't even noticed that property was in there (yet).  It was a blind copy/paste of the Addin class, just to get something working.  Will look into this.

**Comment by devlead** [comment]
@gep13 think we could close this and revisit when Wyam migration is done?

**Comment by gep13** [comment]
@devlead yes, I think you are probably right.

**Pull request closed without merging**